spec/tail_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'filewatch/tail'
+require 'stud/temporary'
+
+describe FileWatch::Tail do
+
+  let(:file_path) { Stud::Temporary.file.path }
+  let(:sincedb_path) { Stud::Temporary.file.path }
+
+  before :each do
+    Thread.new(subject) { sleep 0.5; subject.quit } # force the subscribe loop to exit
+  end
+
+  context "when watching a file" do
+    subject { FileWatch::Tail.new(:sincedb_path => sincedb_path, :start_new_files_at => :beginning) }
```

Comment:
Again, a matter of style, but you can create on subject before the context and then use a let variable (probably named config) who define per context the values.